browser/resources/settings/brave_settings_menu/icons.html

```diff
@@ -0,0 +1,36 @@
+<link rel="import" href="../html/polymer.html">
```

Comment:
Leaving a note here to remember to change this path to point at the path in chromium's resources